src/render/direct3d11/SDL_render_d3d11.c

```diff
@@ -820,16 +787,12 @@ static HRESULT D3D11_CreateDeviceResources(SDL_Renderer *renderer)
     return result;
 }
 
-#if defined(SDL_PLATFORM_WIN32) || defined(SDL_PLATFORM_WINGDK)
```

Comment:
WINGDK

Reply:
This one stays removed; it was only there because WinRT had a separate implementation in a different file, but the (currently) stub function needs to remain for other platforms. Not sure how this compiled for non-WIN GDK before, I assume we _didn't_ compile it.